src/main/java/appeng/client/gui/implementations/GuiPatternTerm.java

```diff
@@ -122,6 +133,17 @@ public void drawFG( final int offsetX, final int offsetY, final int mouseX, fina
 			this.tabProcessButton.visible = false;
 		}
 
+		if( this.container.substitute )
+		{
+			this.substitutionsEnabledBtn.visible = true;
+			this.substitutionsDisabledBtn.visible = false;
+		}
+		else
+		{
+			this.substitutionsEnabledBtn.visible = false;
+			this.substitutionsDisabledBtn.visible = true;
+		}
+
```

Comment:
just thinking as enhancement: 

A class `ToggleGroup` or so, and if you make 1 button visible, all others are disabled via listener. Would be probably interessting in other button groups, so adding a new state is far easier than hard-coding the state.